lib/av/commands/base.rb

```diff
@@ -180,6 +180,27 @@ def parse_param param
         end
         list
       end
+
+      def metadata_rotate degrees
+        add_output_param :'metadata:s:v:0', "rotate=#{degrees}"
+
+        self
+      end
+
+      def filter_metadata_rotate degrees
+        filter_rotate degrees
+
+        if @source
+          current_rotate = identify(@source)[:rotate] || 0
+        else
+          ::Av.log "Source has not been set - assuming current rotation metadata is 0"
+          current_rotate = 0
+        end
+
+        metadata_rotate (current_rotate - degrees) % 360
```

Comment:
I've been doing some experiments with this and whilst I can see why the suggestion is to set the rotation information clockwise, this doesn't actually have the expected effect. I wonder if the rotation metadata is actually intended to convey how to play the video instead of conveying how it was rotated.

If I take a landscape video with no rotate metadata, then rotate it 90 degrees clockwise, and also set the metadata to 90 to show its been rotated 90 degrees - quicktime, vlc, ffplay all show the video upside down - ie 90 degrees rotation + another 90 for (the wrong) correction

Conversely if I take that video and set the metadata to 270, it plays correctly in Quicktime, VLC, ffplay?